docs/advanced_documentation/c-api.md

```diff
@@ -4,29 +4,40 @@ SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynam
 SPDX-License-Identifier: MPL-2.0
 -->
 
-# Use Power Grid Model through C-API
+# Use Power Grid Model through C API
 
 While many users use Python API for Power Grid Model.
-This library also provides a C-API. 
-The main use case of C-API is to integrate Power Grid Model into a non-Python application/library, namely, C, C++, JAVA, C#, etc. 
+This library also provides a C API. 
+The main use case of C API is to integrate Power Grid Model into a non-Python application/library, namely, C, C++, JAVA, C#, etc. 
 
-The C-API consists of a single 
+The C API consists of a single 
 {{ "[header file]({}/power_grid_model_c/power_grid_model_c/include/power_grid_model_c.h)".format(gh_link_head_blob) }}
 and a dynamic library (`.so` or `.dll`) built by a 
 {{ "[cmake project]({}/power_grid_model_c/power_grid_model_c/CMakeLists.txt)".format(gh_link_head_blob) }}.
 Please refer to the [Build Guide](./build-guide.md) about how to build the library.
 
-You can refer to the [C-API Reference](../api_reference/power-grid-model-c-api-reference.rst)
+You can refer to the [C API Reference](../api_reference/power-grid-model-c-api-reference.rst)
 for a detailed documentation of the API. 
 Please also have a look at an 
 {{ "[example]({}/power_grid_model_c_example/main.c)".format(gh_link_head_blob) }}
-of C program to use this C-API.
+of C program to use this C API.
 
-In this documentation, the main design choices and concepts of the C-API are presented.
+In this documentation, the main design choices and concepts of the C API are presented.
+
+## Finding and linking the package
+
+The package can be loaded using the Config mode of the `find_package` CMake command. An
+{{ "[example project]({}/tests/package_tests/CMakeLists.txt)".format(gh_link_head_blob) }} is provided by the
+{{ "[Git project]({})".format(gh_link_head_tree) }}, which is also used for testing the package.
```

Comment:
are you just referring to the root of the repository?

Reply:
yes but i wanted it to be inline and i couldn't find a way to do that. do you have a better suggestion?

Reply:
This is fine.